src/drtsans/tof/eqsans/elastic_reference_normalization.py

```diff
@@ -116,18 +116,18 @@ def normalize_by_elastic_reference_all(
         normalized I(Q2D), normalized I(Q1D), K vector and delta K vector
     """
     # check i_of_q and ref_i_of_q shall have same binning
-    if not verify_same_q_bins(i_of_q_1d, ref_i_of_q_1d, False, tolerance=1e-3):
+    if not verify_same_q_bins(i_1d, ref_i_1d, False, tolerance=1e-3):
         raise RuntimeError("Input I(Q) and elastic reference I(Q) have different Q and wavelength binning")
 
-    wl_vec = np.unique(i_of_q_1d.wavelength)
-    q_vec = np.unique(i_of_q_1d.mod_q)
+    wl_vec = np.unique(i_1d.wavelength)
+    x_vec = np.unique(i_1d.x)
```

Comment:
can  you add a code comment stating that `x` stands either for momentum transfer `q` or azimuthal angle `phi`